deisctl/backend/fleet/journal.go

```diff
@@ -25,6 +25,10 @@ func runJournal(name string) (exit int) {
 		fmt.Fprintf(os.Stderr, "Error retrieving Unit %s: %v", name, err)
 		return 1
 	}
+	if suToGlobal(*u) {
+		fmt.Fprintf(os.Stderr, "Unable to get status for global unit %s. Check the logs on the host using journalctl.", name)
```

Comment:
This should be something like "Unable to get journal for global unit" -- we replaced status in the second sentence, but not the first.